src/gui/tray/Window.qml

```diff
@@ -88,6 +91,11 @@ ApplicationWindow {
             newErrorDialog.text = error
             newErrorDialog.open()
         }
+
+        function onShowFileDetails(accountState, localPath, fileDetailsPage) {
+            console.log(accountState, localPath, fileDetailsPage);

```

Comment:
not sure that is really useful given the existing logs in the c++ part

Reply:
Sorry, was for debugging -- removed